nfs-ganesha-stable/config/definitions/nfs-ganesha-stable.yml

```diff
@@ -76,7 +76,7 @@
       - string:
           name: CEPH_VERSION
           description: "The version of Ceph to specify for installing ceph libraries"
-          default: "12.2.2"
+          default: "12.2.3"
```

Comment:
@ajarr will the latest stable version of luminous still be compatible with 2.5.5?

Reply:
Yes. It should.